torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8562,12 +8562,12 @@ class ForeachRightmostArgType(enum.Enum):
 
 class ForeachSampleInput(SampleInput):
     ref_args: Any
```

Comment:
Now that we are cleaning things, could you remove this ref_args, and instead use `self.args` from the parent class?